src/js/index.js

```diff
@@ -0,0 +1,133 @@
+import createHeader from './components/header/header';
+import createFooter from './components/footer/footer';
+import createTextarea from './components/main/textarea';
+import * as board from './components/main/board';
+
+import '../css/main.scss';
+
+const state = {
+  lang: 'ru',
+  text: '',
+  capsLock: false,
+};
+
+window.addEventListener('beforeunload', () => {
+  localStorage.setItem('language', state.lang);
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  const language = localStorage.getItem('language');
+  if (language !== null) {
+    state.lang = language;
+  }
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  createHeader();
+  createTextarea();
+  board.createKeyBoard(state.lang);
+  createFooter();
+
+  const keyBoard = document.querySelector('.keyboard-list');
+
+  document.addEventListener('keydown', (event) => {
+    if (event.ctrlKey && event.altKey) {
+      if (state.lang === 'ru') {
```

Comment:
state.lang = state.lang === 'ru' ? 'en' : 'ru';

Reply:
Changed to this tipe of function and add switchers for changing language